src/renderer/services/midgard/utils.test.ts

```diff
@@ -252,9 +254,12 @@ describe('services/midgard/utils/', () => {
     })
   })
 
-  describe('toPoolAddresses', () => {
-    it('returns empty list', () => {
-      expect(toPoolAddresses([])).toEqual([])
+  describe('inboundToPoolAddresses', () => {
+    it('adds rune pool address empty list', () => {
+      expect(inboundToPoolAddresses([])).toEqual([RUNE_POOL_ADDRESS])
+    })
+    it('adds two `PoolAddress`es', () => {
+      expect(inboundToPoolAddresses([{ chain: 'BNB', address: 'bnb-address', router: '' }]).length).toEqual(2)
```

Comment:
maybe there is a sense to describe all array ? just to check the content not only a length